tests/test_ugrid.cpp

```diff
@@ -814,6 +814,30 @@ TEST( MeshUgridTest, flow_3d )
   MDAL_CloseMesh( mesh );
 }
 
+TEST( MeshUgridTest, IntegerTimeMesh )
+{
+  std::string path = test_file( "/ugrid/time_integer/simple_time_integer.nc" );
```

Comment:
would it be possible add to this file NC_UINT, NC_INT64, NC_UINT64 extra datasets and check that the values read from them are also correct?